back/src/Guide/Database/Get.hs

```diff
@@ -188,6 +178,46 @@ getItemsByCategory catId (arg #deleted -> deleted) = do
   itemUids <- lift $ HT.statement (catId,deleted) (Statement sql encoder decoder False)
   traverse getItem itemUids
 
+-- | Get 'pros_order' or 'cons_order' of item..
+getItemTraitsOrder :: Uid Item -> TraitType -> ExceptT DatabaseError Transaction [Uid Trait]
+getItemTraitsOrder itemId traitType = do
+  let sql = case traitType of
+        Pro -> [r|
+          SELECT pros_order
+          FROM items
+          WHERE uid = $1
+          |]
+        Con -> [r|
+          SELECT cons_order
+          FROM items
+          WHERE uid = $1
+          |]
+      encoder = uidParam
+      decoder = HD.singleRow uidsColumn
+  lift $ HT.statement itemId (Statement sql encoder decoder False)
+
+-- | Get items by trait
+getItemIdByTraitMaybe :: Uid Trait -> ExceptT DatabaseError Transaction (Maybe (Uid Item))
+getItemIdByTraitMaybe traitId = do
+  let sql = [r|
+        SELECT item_uid
+        FROM traits
+        WHERE uid = $1
+        |]
+      encoder = uidParam
+      decoder = HD.rowMaybe $ uidColumn
+  lift $ HT.statement traitId (Statement sql encoder decoder False)
+
+-- | Get items by trait
+--
+-- Throw an error when 'Uid Trait' not found.
```

Comment:
Which error? It's better to mention that in the Haddock.

`-- Can throw 'TraitNotFound'.`